### PR TITLE
nits

### PR DESCRIPTION
**Pull request by thereisnogabe**
mostly variable namings and fail-fast handling

### CODE REVIEW

File: plugin/healthcheck.go [hunk 1]
```diff
@@ -30,20 +32,22 @@ func (*HealthcheckPlugin) Version() string {
...
 
 	cfg, ok := env.Config.(map[string]interface{})
-	if ok {
-		_port, ok := cfg["port"].(string)
-		if ok {
-			port = _port
-			return nil
-		}
+	if !ok {
+		return errors.New("env config not in known format")
```
Comment (v-stickykeys) [inline plugin/healthcheck.go:43]: I don't think we want to error here. We can just use default port and log warning?

I'm guessing error here would stop the process?

File: server.go [hunk 2]
```diff
@@ -47,30 +47,30 @@ func StartServer(port string, ipfs coreiface.CoreAPI) {
 		fmt.Println("Healthcheck server shutting down...")
 		close(idleConnsClosed)
 		if err := server.Shutdown(context.Background()); err != nil {
-			log.Printf("Healthcheck server error on Shutdown: %v", err)
+			log.Printf("Healthcheck server error on Shutdown: %+v", err)
```
Comment (v-stickykeys) [inline server.go:50]: What does the plus do?

File: server.go [hunk 3]
```diff
@@ -99,10 +99,10 @@ func healthcheckHandler(w http.ResponseWriter, r *http.Request) {
...
 		w.WriteHeader(http.StatusInternalServerError)
-		_status = &status{Message: "Health check failed"}
+		healthCheck = &status{Message: "Health check failed"}
 	} else {
-		_status = &status{Message: "Health check succeeded"}
+		healthCheck = &status{Message: "Health check succeeded"}
 	}
 
-	fmt.Fprintf(w, _status.Message)
+	_, _ = fmt.Fprintf(w, healthCheck.Message)
```
Comment (v-stickykeys) [inline server.go:107]: I guess this catches any error returned from `.Message`?

File: go.sum [hunk 4]
```diff
@@ -664,7 +664,6 @@ github.com/libp2p/go-libp2p-yamux v0.5.1/go.mod h1:dowuvDu8CRWmr0iqySMiSxK+W0iL5
 github.com/libp2p/go-maddr-filter v0.0.1/go.mod h1:6eT12kSQMA9x2pvFQa+xesMKUBlj9VImZbj3B9FBH/Q=
 github.com/libp2p/go-maddr-filter v0.0.4/go.mod h1:6eT12kSQMA9x2pvFQa+xesMKUBlj9VImZbj3B9FBH/Q=
 github.com/libp2p/go-maddr-filter v0.0.5/go.mod h1:Jk+36PMfIqCJhAnaASRH83bdAvfDRp/w6ENFaC9bG+M=
-github.com/libp2p/go-maddr-filter v0.1.0 h1:4ACqZKw8AqiuJfwFGq1CYDFugfXTOos+qQ3DETkhtCE=
```
Comment (v-stickykeys) [inline go.sum:667]: Is this go.sum file meant to change? I guess I was seeing it as equivalent to a package-lock file. And given we are using the same version of go it should not change correct?

Comment (glcohen) [reply go.sum:667]: it looks like when I did `go build` it pulled a more updated version of the dep.

File: server.go (hunk 2 above)
Comment (glcohen) [reply server.go:51]: adds field names https://pkg.go.dev/fmt#hdr-Printing
I can remove, I usually keep it there for more info, `err.String()` could work too - let me know

**Pull request merged**